### ISSUE TITLE
Scrollwheel Broken

### ISSUE THREAD

**Issue by camas002** [cmooref17/Lethal-Company-TooManyEmotes#59]
I was just playing with the mod before it updated to 1.9.0 and my scroll wheel does not scroll anymore. Maybe with 1.9.0 update it now conflicts with other mods, but it started happening as soon as I updated to 1.9.0. After disabling TooManyEmotes, my scroll wheel works again.

**Comment by idiotdoomspiral** [comment]
I also have this issue.  Reverting to the previous version fixed it.

**Comment by pinksharebear** [comment]
I also just had this issue. Didn't have any problems last night and then tonight right after the update the scroll wheel will not work in the main game. turned this one off, but looking forward to an update for it to be fixed as this is one of our favorite mods

**Comment by cmooref17** [comment]
Does this break the scroll wheel for everything? Swapping hot bar slots? Or is it only the mod related scrolling, like in the emote menu?

**Comment by pinksharebear** [comment]
> Does this break the scroll wheel for everything? Swapping hot bar slots? Or is it only the mod related scrolling, like in the emote menu?

I could not select anything in the slot bar inventory for the main game or for the add on reserve walkie/flashlight/weapon slots. The scroll wheel worked just fine in the emote menu.

**Comment by KinkyKluts** [comment]
made an account to comment, i did some testing and the scroll wheel for cycling inventory is broken and it is not a compatibility issue, because installing just it still makes it break

**Comment by pinksharebear** [comment]
Yep, exactly as KinkyKluts says.

**Comment by TinyEmbar** [comment]
When I read the console that boots with the game it spat out something like "Emote wheel +180" and "Emote wheel -180" but i am no longer sure since I reverted.

**Comment by idiotdoomspiral** [comment]
> Does this break the scroll wheel for everything? Swapping hot bar slots? Or is it only the mod related scrolling, like in the emote menu?

Yep, seems so.  The hot bar couldn't be cycled, even when changing binds.

**Comment by Jeaterbeater** [comment]
hey! made an account to comment, mine and my husbands scroll wheels are also broken. reverting to old update or uninstalling the mod fixes it. even when we change the binds the scrolling through the hot bar still doesn't work. love this mod hope it gets fixed soon!

**Comment by Lordmau5** [comment]
Thought I'd add some extra info to this:

Scrolling in the Terminal also doesn't work when 1.9.0 of this mod is installed

**Comment by cmooref17** [comment]
Sorry about this! This has been fixed, and the update has been pushed. It should be working now on 1.9.1.
